<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SampleAdder</class>
 <widget class="QDialog" name="SampleAdder">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>421</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>افزودن فرد جدید</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>351</width>
     <height>151</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="title">
    <string>مشخصات</string>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="3" column="0">
     <widget class="QLabel" name="label_name">
      <property name="text">
       <string>نام</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="lineEdit_name"/>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="lineEdit_2"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_lastName">
      <property name="text">
       <string>نام خانوادگی</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEdit_id">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_id">
      <property name="text">
       <string>شناسه</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>210</y>
     <width>206</width>
     <height>30</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="pushButton_addSampleCancel">
      <property name="text">
       <string>انصراف</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_addSampleTakeSamples">
      <property name="text">
       <string>افزودن</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
